web/src/input_pill.ts

```diff
@@ -438,6 +438,18 @@ export function create<T>(opts: InputPillCreateOptions<T>): InputPillContainer<T
             }
         });
 
+        store.$parent.on("dblclick", (e) => {
+            e.stopPropagation();
+            const $pill = $(e.target)?.closest(".pill");
+            const text = $pill?.find(".pill-label")?.text()?.trim() ?? "";
```

Comment:
I'm not sure the `text()` value of the pill will always be what you need to recreate the pill in other inputs, like the compose  box user picker, or the stream subscribers picker that supports users/groups/streams. 

So we might need to rework `InputPillItem` to have a `text_value` field that we populate with the exact original text used to create the pill, maybe as a preparatory refactor commit?

Reply:
This might be a lot easier as a refactor now that `input_pill.ts` is TypeScript.

Reply:
I have tried in all the pill use cases, this `text` value does the job.